### ISSUE TITLE
Cannot compile package on Lazarus trunk

### ISSUE THREAD

**Issue by Alexey-T** [BerndGabriel/HtmlViewer#289]
Lazarus 2.1.0 r63996 FPC 3.0.4 x86_64-linux-gtk2

11.9 branch

```
Compile package FrameViewer09 11.0: Exit code 1, Errors: 2, Warnings: 50, Hints: 2
htmlmisc.pas(326,36) Warning: Symbol "CM_WININICHANGE" is not implemented
htmlmisc.pas(893,10) Warning: Function result does not seem to be set
htmlmisc.pas(902,10) Warning: Function result does not seem to be set
htmlmisc.pas(1060,10) Warning: Function result does not seem to be set
htmlmisc.pas(1069,10) Warning: Function result does not seem to be set
htmlmisc.pas(1078,10) Warning: Function result does not seem to be set
htmlmisc.pas(1087,10) Warning: Function result does not seem to be set
htmlmisc.pas(1143,10) Warning: Function result does not seem to be set
htmlmisc.pas(1152,10) Warning: Function result does not seem to be set
htmlmisc.pas(1162,10) Warning: Function result does not seem to be set
htmlmisc.pas(1171,10) Warning: Function result does not seem to be set
htmlmisc.pas(1180,10) Warning: Function result does not seem to be set
htmlmisc.pas(1189,10) Warning: Function result does not seem to be set
htmlmisc.pas(1198,10) Warning: Function result does not seem to be set
HtmlGlobals.pas(876,3) Warning: Variable "SystemPalette16" read but nowhere assigned
htmlgif1.pas(1516,20) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(1550,20) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(1679,14) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(1892,24) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(1895,24) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(1898,37) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2034,18) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2043,21) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2062,24) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2063,29) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2065,27) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2068,18) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2068,30) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2070,39) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2125,22) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2127,25) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2128,26) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2130,29) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(2139,27) Warning: Converting pointers to signed integers may result in wrong comparison results and range errors, use an unsigned type instead.
htmlgif1.pas(416,3) Note: Local variable "GIF_ErrorCode" is assigned but never used
HtmlImages.pas(884,5) Note: Local variable "IconInfo" not used
HtmlImages.pas(1267,48) Warning: Symbol "Handle" is deprecated
HTMLUn2.pas(404,15) Warning: An inherited method is hidden by "SetVisible(Boolean);"
StyleUn.pas(3820,16) Warning: Local variable "S1" of a managed type does not seem to be initialized
StylePars.pas(67,5) Warning: Constructor should be public
StylePars.pas(1299,33) Warning: Variable "Value" of a managed type does not seem to be initialized
HTMLUn2.pas(2655,34) Warning: Symbol "Handle" is deprecated
HTMLUn2.pas(2791,49) Warning: Symbol "Handle" is deprecated
HTMLSubs.pas(759,5) Warning: Constructor should be public
UrlConn.pas(957,18) Warning: Symbol "HRSRC" is deprecated
htmlview.pas(312,14) Warning: An inherited method is hidden by "GetCursor:SmallInt;"
htmlview.pas(352,15) Warning: An inherited method is hidden by "SetBorderStyle(THTMLBorderStyle);"
htmlview.pas(2562,10) Warning: Symbol "GetTickCount" is deprecated: "Use GetTickCount64 instead"
htmlview.pas(2564,18) Warning: Symbol "GetTickCount" is deprecated: "Use GetTickCount64 instead"
htmlview.pas(3417,50) Warning: Symbol "Handle" is deprecated
framview.pas(3376,35) Warning: Conversion between ordinals and pointers is not portable
framview.pas(3391,11) Warning: Conversion between ordinals and pointers is not portable
vwPrint.pas(114,47) Error: Identifier not found "PDeviceModeW"
vwPrint.pas(114,88) Error: Error in type definition

```

**Comment by Alexey-T** [comment]
I attach fixed file, pls use it (code copied from FPC 3.3 trunk)

[vwPrint.zip](https://github.com/BerndGabriel/HtmlViewer/files/5374523/vwPrint.zip)

**Comment by BerndGabriel** [comment]
Will be fixed in Version 11.10
